waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/dto/BriefAssetDescription.scala

```diff
@@ -2,15 +2,14 @@ package com.wavesplatform.dex.grpc.integration.dto
 
 import com.wavesplatform.transaction.assets.exchange.AssetPair
 
-case class BriefAssetDescription(name: String, decimals: Int, hasScript: Boolean)
+case class BriefAssetDescription(name: String, decimals: Int, hasScript: Boolean = false)
```

Comment:
I looked through the code and it seems ok for BriefAssetDescription now, but generally I think it's not a good idea to use a parameter's default value is the main constructor. It's easy to forget about it in places, where we need to specify a non-default value.

Reply:
Done